api/config/service/topology/v1/topology.proto

```diff
@@ -15,4 +15,14 @@ message Config {
 message Cache {
   // The time to live (ttl) for an item in cache, the default is 2 hours (7200 seconds).
   google.protobuf.Duration ttl = 1 [ (validate.rules).duration.gt.seconds = 1 ];
+
+  // Set the batch insert size for setCache() operations. By default this is set to 1.
+  // Depending on your database resource configuration and the number of items to cache,
+  // you can tune this value to improve write performance.
+  int32 batch_insert_size = 2;
+
+  // Defaults to 10 seconds.
```

Comment:
curious, is it possible to get the default value with the validation rule on the field?

Reply:
oh interesting i didn't know the validation had default support, i wouldn't want to force users to use 10seconds as the floor tho. But let me check it out.

Reply:
sorry, i meant will the validation prevent someone from entering no value, in which case there is no default anyways

Reply:
Ah i got ya my bad, yes let me remove the validation.